core/src/main/kotlin/api/ProductsRoute.kt

```diff
@@ -23,6 +23,7 @@ import io.github.smiley4.ktorswaggerui.dsl.routing.delete
 import io.github.smiley4.ktorswaggerui.dsl.routing.get
 import io.github.smiley4.ktorswaggerui.dsl.routing.patch
 import io.github.smiley4.ktorswaggerui.dsl.routing.post
+import io.github.smiley4.ktorswaggerui.dsl.routing.put
```

Comment:
Commit message nits:
* plural "endpoints" in title, and maybe change to "Add endpoints to manage product groups"
* "2 new" is redundant.